### ISSUE TITLE
ColorMap/ColorMask Textures with Transparency Results in Excessive Bloom

### ISSUE THREAD

**Issue by yoshiyoshyosh** [Yellow-Dog-Man/Resonite-Issues#730]
### Describe the bug?

Color Splat and Color Mask materials use a `ColorMap` / `ColorMask` texture, respectively. On Linux Native, when part of the texture is transparent (or, in the case of `ColorMask`, if the texture is `null`), then said part of the texture will be all-black with excessive bloom. This does not change if the `AlbedoTexture` or `EmissiveMap` is changed.

### To Reproduce

1. Use Linux Native
2. Spawn a `Color Splat` or `Color Mask` material
3. Drop a texture that has transparent parts (but is not completely transparent) in the `ColorMap` or `ColorMask` texture slot

An item that showcases this behavior using a `ColorMap` material is `resrec:///U-yosh/R-ead4e8cc-8092-483e-9c4a-993d655934c2`

### Expected behavior

Excessive Bloom does not occur, as is the case on Windows.

### Screenshots

Putting a 4x4 texture with 1 non-transparent pixel in a Color Splat material's `ColorMap`
![2023-11-18_11-55-48](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/36683489/e9e23762-5441-4ea4-9934-a46ba7e6e939)

Color Mask material orb with a 4x4 texture with 1 non-transparent pixel on the `ColorMask` texture
![2023-11-18_12-13-51](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/36683489/66b44bb7-45ad-4dc4-9944-5d71325714e3)

Color Mask material orb with a null `ColorMask`
![2023-11-18_12-16-51](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/36683489/2692488d-8c09-4fdd-87e8-82c24e42a9ef)

### Resonite Version Number

2023.11.15.1158

### What Platforms does this occur on?

Linux

### Log Files

[Color Map Log](https://gist.github.com/yoshiyoshyosh/0cab0a78bf2f5ef34aec54ad073f6070) (putting the material on a box, image 1)

[Color Mask Log](https://gist.github.com/yoshiyoshyosh/11e4b5c9dab6dd369046aa34ff0a97f2) (spawning a color mask orb with null color mask, image 3)

### Additional Context

this is most likely a unity issue

### Reporters

yosh

**Comment by yoshiyoshyosh** [comment]
> Is this issue still relevant even when running the Windows build through Proton,[@yoshiyoshyosh](https://github.com/yoshiyoshyosh)? If not, this can probably be closed for now as the native Linux client has been deprecated indefinitely.

no. it is also a unity issue, so it'll most likely go away once we're off unity

**Comment by shiftyscales** [comment]
Thank you. :)
